apps/frontend/src/components/Dashboard/Post.vue

```diff
@@ -160,11 +160,14 @@
             <div class="mt-4 text-gray-600">
                 <!-- <p class="my-2 break-words">{{ item.post.body }}</p>-->
                 <div class="my-2 break-words whitespace-pre-wrap">
-                    <p v-if="!readMore && item.post.body.length > 200">
-                        {{ item.post.body.slice(0, 200) }}
+                    <p
+                        v-if="!readMore && amount_lines + 1 > 5"
```

Comment:
move v-if to span so you do not have to vue directives bound to one element (v-if and v-for)